scripts/ScriptUtilities.psm1

```diff
@@ -12,13 +12,13 @@ $InformationPreference = "Continue"
 
 $RepoRoot = $(Resolve-Path $PSScriptRoot\..).Path
 $Platform = "AnyCPU"
-$SourceRoot = "$RepoRoot\src"
-$JsonSchemaPath = "$SourceRoot\Sarif\Schemata\sarif-schema.json"
-$BuildPropsPath = "$SourceRoot\build.props"
-$BuildRoot = "$RepoRoot\bld"
-$BinRoot = "$BuildRoot\bin"
+$SourceRoot = Join-Path $RepoRoot "src"
+$JsonSchemaPath = Join-Path (Join-Path (Join-Path $SourceRoot "Sarif") "Schemata") "sarif-schema.json"
```

Comment:
>Join [](http://example.com/codeflow?start=18&length=4)

Can we investigate upgrading powershell in our build environment so that we can use a simplified pattern here?

Feel free to investigate this as a filed issue/follow-up item.

Reply:
A quick check I clicked the "about this task" link of the PowerShell task:
[https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/powershell?view=azure-devops)

does not seem to able to change the "2" to other number